github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -2020,7 +2026,8 @@ presubmits:
             memory: 4Gi
         securityContext:
           privileged: true
-  - always_run: true
+  - always_run: false
```

Comment:
it seems that as part of the rebase this went wrong, 1.29 should be always_run=true
will go over them all and fix

Reply:
can you create a PR fixing this?

Reply:
sure on it

Reply:
https://github.com/kubevirt/project-infra/pull/3205
please take a look